src/CarbonAware.WebApi/src/Filters/HttpResponseExceptionFilter.cs

```diff
@@ -9,62 +9,75 @@ namespace CarbonAware.WebApi.Filters;
 public class HttpResponseExceptionFilter : IExceptionFilter
 {
     private ILogger<HttpResponseExceptionFilter> _logger;
-    private static readonly ActivitySource _activitySource = new ActivitySource(nameof(HttpResponseExceptionFilter));
+    private IConfiguration _config;
 
     private static Dictionary<string, int> EXCEPTION_STATUS_CODE_MAP = new Dictionary<string, int>()
     {
         { "ArgumentException", (int)HttpStatusCode.BadRequest },
         { "NotImplementedException", (int)HttpStatusCode.NotImplemented },
     };
 
-    public HttpResponseExceptionFilter(ILogger<HttpResponseExceptionFilter> logger)
+    public HttpResponseExceptionFilter(ILogger<HttpResponseExceptionFilter> logger, IConfiguration configuration)
```

Comment:
Using IConfiguration directly isn't the right way to do this when dependency injection is available.  Instead, inject an IOptionMonitor&lt;CarbonAwareVariablesConfiguration&gt; object.